src/applications/disability-benefits/new-686/config/form.js

```diff
@@ -0,0 +1,56 @@
+// import fullSchema from 'vets-json-schema/dist/21-686-schema.json';
```

Comment:
If this is unnecessary, should it be deleted instead of commented out?

Reply:
This schema will be used in the future, this branch is just for us to test out building the form from scratch and if we decide to use it we will also be using this schema in the future.